style/components/_requests.scss

```diff
@@ -1,36 +1,20 @@
 .requests {
-
-  position: absolute;
-  top: 55px;
-  left: 0;
-  bottom: 0;
-  right: 0;
   overflow-y: auto;
-
-  .request-inner {
-    padding: 5px;
-    overflow: hidden;
-  }
+  overflow-x: hidden;
+  height: calc(100% - 55px); //55px is height of search bar (header)
 
   .request {
-    position: absolute;
-    left: 0;
-    right: 0;
+    position: relative; //absolutely-positioned context menu needs to be attached to request
     border-bottom: 2px solid $lightgray-light;
+    padding: 2px 0;
```

Comment:
Since requests didn't have a height before, is rather not add then to this pr if I don't have to